exonum-light-client/src/main/java/com/exonum/client/response/BlocksResponse.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.client.response;
+
+import java.util.List;
+import lombok.Value;
+
+@Value
+public class BlocksResponse {
+  /**
+   * Blockchain blocks in descending order by height.
+   * It is allowed to be empty if no blocks found.
+   * It may contain gaps.
```

Comment:
I'd remove this line and change the first one to
`Blockchain blocks in descending order (not necessarily continuous) by height.`
If you think the current is better that's okay as well, but for me it took a moment to understand what those gaps are.